components/expenses/ExpenseDetails.js

```diff
@@ -66,6 +70,10 @@ class ExpenseDetails extends React.Component {
       ...this.state.expense,
       [attr]: value,
     };
+    if (expense['type'] !== 'RECEIPT' && expense['type'] !== 'INVOICE') {
+      console.error(this.props.intl.formatMessage(this.messages['expenseTypeMissing']));
```

Comment:
Writing an internationalized message to the console is not something do:
- if it's for developers, they don't need internationalization
- if it's for the user, they will not see it

Reply:
Removed